### PR TITLE
Add org.kde.skrooge

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
As requested here https://twitter.com/vosaxalo/status/1059159975518445569
Sent an email to the devs to see if they are interested in co-maintain the Flatpak package. The mimetypes are not exported, talked about that on the email I have sent to the devs.

### CODE REVIEW

**Comment by TingPing** [comment]
>  The mimetypes are not exported

In the mean time you can just patch that.

File: org.kde.skrooge.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          - AUTOMAKE="automake --foreign" autoreconf -vfi
+
+  - name: qca
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DQT4_BUILD=OFF
+      - -DQCA_SUFFIX=qt5
+    sources:
+      - type: archive
+        url: http://download.kde.org/stable/qca/2.1.3/src/qca-2.1.3.tar.xz
```
Comment (TingPing) [inline org.kde.skrooge.yml]:
```suggestion
        url: https://download.kde.org/stable/qca/2.1.3/src/qca-2.1.3.tar.xz
```

File: org.kde.skrooge.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+base-version: '5.11'
+command: skrooge
+rename-icon: skrooge
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+cleanup:
```
Comment (TingPing) [inline org.kde.skrooge.yml]: You need to manually delete the baseapp files with `cleanup-commands` I believe.

**Comment by nedrichards** [comment]
bot, build org.kde.skrooge

**Comment by bilelmoussaoui** [comment]
Builds perfectly. https://flathub.org/builds/#/builders/2/builds/6028 Still would like to get those patches for the mimetypes. We should definitely get something like `rename-mimetype`

**Comment by nedrichards** [comment]
bot, build org.kde.skrooge

File: org.kde.skrooge.yml (hunk 2 above)
Comment (barthalion) [reply org.kde.skrooge.yml]: I think `cleanup` also handles it fine.

**Comment by barthalion** [comment]
Builds fine, let's add mimetype renames later.

Repository has been created: https://github.com/flathub/org.kde.skrooge

Thanks!

**Pull request closed without merging**